htp/htp_config_private.h

```diff
@@ -343,6 +343,9 @@ struct htp_cfg_t {
     /** How many layers of compression we will decompress (0 => no limit). */
     int response_decompression_layer_limit;
 
+    /** How many layers of compression we will decompress (0 => no lzma). */
+    int response_lzma_layer_limit;
```

Comment:
I think this will break the ABI. Please append the new field so the layout in memory doesn't change

Reply:
Done in next PR